checker/check.py

```diff
@@ -0,0 +1,254 @@
+from django.conf import settings
+from django.db import models
+
+from github import Github
+from cloudfoundry_client.client import CloudFoundryClient
+from datetime import datetime
+import pytz
+import yaml
+import csv
+from .models import PipelineApp, PipelineEnv
+
+import logging
+
+log = logging.getLogger(__name__)
+logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)
+
+
+def get_pipeline_configs(repo):
+    yaml_file_list=[]
+    for content_file in repo.get_contents(""):
+        if ".yaml" in content_file.path:
+            yaml_file_list.append(content_file.path)
+    return yaml_file_list  
+
+
+def get_app_config_yaml(repo, config_file):
+    config_text = repo.get_contents(config_file).decoded_content.decode()
+    config_yaml = yaml.load(config_text, Loader=yaml.FullLoader)
```

Comment:
```suggestion
    config_yaml = yaml.safe_load(config_text)
```
loading yaml is unsafe unless you use the `safe_loader`

Reply:
OK, done. 👍